src/main/java/frc/robot/commands/AutoScore.java

```diff
@@ -0,0 +1,53 @@
+package frc.robot.commands;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitUntilCommand;
+import frc.robot.subsystems.Intake;
+import frc.robot.subsystems.Lift;
+import frc.robot.subsystems.TagLimelight;
+import frc.robot.subsystems.Lift.LiftPosition;
+import java.util.HashMap;
+import com.pathplanner.lib.PathConstraints;
+import com.pathplanner.lib.PathPlanner;
+import com.pathplanner.lib.PathPlannerTrajectory;
+import com.pathplanner.lib.commands.FollowPathWithEvents;
+
+import frc.robot.Cal;
+import frc.robot.subsystems.drive.DriveSubsystem;
+
+public class AutoScore extends SequentialCommandGroup{
+    private DriveSubsystem drive;
+    private PathPlannerTrajectory trajInit =
+        PathPlanner.loadPath(
+    "InitScoreAndGetGamePiece",
+        new PathConstraints(Cal.PLACEHOLDER_DOUBLE, Cal.PLACEHOLDER_DOUBLE));
+    private PathPlannerTrajectory trajCharge =
+        PathPlanner.loadPath(
+    "ScoringLocToChargeStation",
+        new PathConstraints(Cal.PLACEHOLDER_DOUBLE, Cal.PLACEHOLDER_DOUBLE));
+    private HashMap<String, Command> eventMap = new HashMap<>();
+    private boolean isFirstPath = true;
+    
+    public AutoScore(Lift lift, Intake intake, DriveSubsystem drive, TagLimelight tagLimelight){
+        eventMap.put("deployIntake", new InstantCommand(()->{intake.setDesiredDeployed(true);}, intake));
+        eventMap.put("closeIntake", new InstantCommand(()->{intake.setDesiredDeployed(false);}, intake));
+        addCommands(
+            new InstantCommand(lift::manualPrepScore, lift),
+            new WaitUntilCommand(()->lift.atPosition(LiftPosition.MANUAL_PREP_SCORE)),
```

Comment:
This unfortunately isn't a position, I think Asher has the correct position in a PR that just merged

Reply:
So you can just fetch and rebase onto origin/main

Reply:
wdym by this? Just a little confused by this

Reply:
he means that there is a position for manual prep score that asher pushed recently, and it should be replaced with that